scripts/install_python.sh

```diff
@@ -102,8 +102,9 @@ if [ $DO_INSTALL == "yes" ]; then
     mkdir $TMP_27
     tar -xzf $SOURCE_27 -C $TMP_WORKSPACE
     cd $TMP_27
-    ./configure --prefix="$INSTALL_PREFIX"
+    ./configure --enable-shared --prefix="$INSTALL_PREFIX"
     make && make altinstall
+    sudo ldconfig
     sudo python2.7 -m ensurepip
```

Comment:
I'm testing on Ubuntu and it seems that pip wasn't installed. Strangely, I'd assume that this line should have made sure it's installed? I had to use `sudo apt install python-pip`. I guess this is only run if python 2.7 isn't already installed. Probably easiest to just add python-pip to the list of python 2.7 packages to install in the Build Readme